packages/react-app/src/components/ModalIntro.jsx

```diff
@@ -1,21 +1,20 @@
 import React, { useState } from "react";
```

Comment:
I don't think I saw a modal intro when I used the demo. What does this component do?

Reply:
This is the modal that pops up when you click the info button.